package.json

```diff
@@ -59,12 +60,12 @@
     "-- CI SCRIPTS ": "-----------------------------------------------------------------------------------------------",
     "build": "./scripts/build/build-ci.sh",
     "ci-compile": "node scripts/ci/check-versions.js && node scripts/build/cli npmCiModules && npm run lint && npm run unit-nginx && npm run unit-haproxy && npm run unit-haproxy-healthcheck && npm run unit-cht-deploy && npm run build && npm run integration-api && npm run unit",
-    "ci-integration-all": "mocha --config tests/integration/.mocharc-all.js",
-    "ci-integration-all-k3d": "mocha --config tests/integration/.mocharc-k3d.js",
-    "ci-integration-sentinel": "mocha --config tests/integration/.mocharc-sentinel.js",
-    "ci-integration-sentinel-k3d": "mocha --config tests/integration/.mocharc-sentinel-k3d.js",
-    "ci-webdriver-default": "wdio run ./tests/e2e/default/wdio.conf.js",
-    "ci-webdriver-default-mobile": "wdio run ./tests/e2e/default-mobile/wdio.conf.js",
+    "ci-integration-all": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && mocha --config tests/integration/.mocharc-all.js",
+    "ci-integration-all-k3d": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && mocha --config tests/integration/.mocharc-k3d.js",
+    "ci-integration-sentinel": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && mocha --config tests/integration/.mocharc-sentinel.js",
+    "ci-integration-sentinel-k3d": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && mocha --config tests/integration/.mocharc-sentinel-k3d.js",
+    "ci-webdriver-default": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} && wdio run ./tests/e2e/default/wdio.conf.js",
+    "ci-webdriver-default-mobile": "export VERSION=${VERSION:-$(node ./scripts/build/get-version.js)} wdio run ./tests/e2e/default-mobile/wdio.conf.js",
```

Comment:
Are these even needed anymore? I think with the change of how we set versions locally, these are not needed.

Reply:
Locally, with just the change in `scripts/build/versions.js` I can run the local command followed by the ci command, unchanged, and I get to use the correct images.

Reply:
Ohhh I thought that I needed to export the `VERSION` again to be able to use the correct images, but I just tested it and it worked perfectly, thanks Diana ⭐